clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/BasicAmazonDeployDescription.groovy

```diff
@@ -132,6 +132,13 @@ class BasicAmazonDeployDescription extends AbstractAmazonCredentialsDescription
    */
   Boolean unlimitedCpuCredits
 
+  /**
+   * When set to true, the created server group will be configured with Nitro Enclaves enabled
+   * This is a Launch Template only feature
+   * * https://docs.aws.amazon.com/enclaves/latest/user/nitro-enclave.html
+   */
+  Boolean enableEnclave = false
```

Comment:
@iandelahorne remove the default here in order to set enclave options only when specified in request.